TEveEventDisplay/fcl/prolog.fcl

```diff
@@ -62,8 +60,7 @@ TEveEventDisplayCosmics : {
   }   
 
 TEveEventDisplayCaloOnly : {
```

Comment:
Attempted, due to use of the filler this hasn't been as nice as it could have been since I had to have all the fields of the filler for each instance (otherwise I see errors saying ``` ====================================================================================================
  
  !! The following modules have been misconfigured: !!
  
  ----------------------------------------------------------------------------------------------------
  
  
  Module label: TEveEventDisplayHelix
  module_type : TEveEventDisplay
  
  Any parameters prefaced with '#' are optional.
  Missing parameters:
  
        filler: {
   -       CaloClusterCollection: <art::InputTag>
        }
```
etc.